README.md

```diff
@@ -13,7 +13,7 @@ See the `standalone/example/` folder for a fully functional example.
 
 ## Dependency
 
-It's recommended that you include [Essential](link eventually) instead of adding it yourself.
+It's recommended that you include [Essential](https://essential.gg/) instead of adding it yourself.
```

Comment:
This link is meant to direct to either a template or repo on how to include essential as a dependency. Currently this is waiting on some other things so this isn't ready yet.